patchwork-registries/src/main/java/net/minecraftforge/registries/ForgeRegistries.java

```diff
@@ -109,42 +102,49 @@
-		ACTIVITIES = wrap("activity", Activity.class);
+		POTIONS = wrap("mob_effect", StatusEffect.class);
 		BIOMES = wrap("biome", Biome.class);
-		BIOME_PROVIDER_TYPES = wrap("biome_source_type", BiomeSourceType.class);
-		TILE_ENTITIES = wrap("block_entity_type", BlockEntityType.class);
-		WORLD_CARVERS = wrap("carver", Carver.class);
-		CHUNK_GENERATOR_TYPES = wrap("chunk_generator_type", ChunkGeneratorType.class);
-		CHUNK_STATUS = wrap("chunk_status", ChunkStatus.class);
-		wrap("custom_stat", Identifier.class);
-		DECORATORS = wrap("decorator", Decorator.class);
-		wrap("dimension_type", DimensionType.class);
+		SOUND_EVENTS = wrap("sound_event", SoundEvent.class);
+		POTION_TYPES = wrap("potion", Potion.class);
 		ENCHANTMENTS = wrap("enchantment", Enchantment.class);
 		ENTITIES = wrap("entity_type", EntityType.class);
-		FEATURES = wrap("feature", Feature.class);
-		FLUIDS = wrap("fluid", Fluid.class);
-		MEMORY_MODULE_TYPES = wrap("memory_module_type", MemoryModuleType.class);
+		TILE_ENTITIES = wrap("block_entity_type", BlockEntityType.class);
+		PARTICLE_TYPES = wrap("particle_type", ParticleType.class);
 		CONTAINERS = wrap("menu", ContainerType.class);
-		POTIONS = wrap("mob_effect", StatusEffect.class);
 		PAINTING_TYPES = wrap("motive", PaintingMotive.class);
-		PARTICLE_TYPES = wrap("particle_type", ParticleType.class);
-		POI_TYPES = wrap("point_of_interest_type", PointOfInterestType.class);
-		POTION_TYPES = wrap("potion", Potion.class);
 		RECIPE_SERIALIZERS = wrap("recipe_serializer", RecipeSerializer.class);
+		STAT_TYPES = wrap("stat_type", StatType.class);
+
+		// Villages
+		PROFESSIONS = wrap("villager_profession", VillagerProfession.class);
+		POI_TYPES = wrap("point_of_interest_type", PointOfInterestType.class);
+		MEMORY_MODULE_TYPES = wrap("memory_module_type", MemoryModuleType.class);
+		SENSOR_TYPES = wrap("sensor_type", SensorType.class);
+		SCHEDULES = wrap("schedule", Schedule.class);
+		ACTIVITIES = wrap("activity", Activity.class);
+
+		// Worldgen
+		WORLD_CARVERS = wrap("carver", Carver.class);
+		SURFACE_BUILDERS = wrap("surface_builder", SurfaceBuilder.class);
+		FEATURES = wrap("feature", Feature.class);
+		DECORATORS = wrap("decorator", Decorator.class);
+		BIOME_PROVIDER_TYPES = wrap("biome_source_type", BiomeSourceType.class);
+		CHUNK_GENERATOR_TYPES = wrap("chunk_generator_type", ChunkGeneratorType.class);
+		CHUNK_STATUS = wrap("chunk_status", ChunkStatus.class);
+
+		// TODO: Patchwork Project, check these
+		wrap("dimension_type", DimensionType.class);
+		wrap("custom_stat", Identifier.class);
 		wrap("recipe_type", RecipeType.class);
 		wrap("rule_test", RuleTestType.class);
-		SCHEDULES = wrap("schedule", Schedule.class);
-		SENSOR_TYPES = wrap("sensor_type", SensorType.class);
-		SOUND_EVENTS = wrap("sound_event", SoundEvent.class);
-		STAT_TYPES = wrap("stat_type", StatType.class);
-		wrap("structure_feature", StructureFeature.class);
+		/*wrap("structure_feature", StructureFeature.class);
```

Comment:
Why comment this out when it wasn't previously?

Reply:
These types don't have a registry in Forge.
In particular, StructureFeature extends Feature, so StructureFeature cannot be a registry base class, as this violates forge's registry rule: a new registry type should not be a sub class of an existing registry type

Please correct me if these are not true.